signature-react-app/README.md

```diff
@@ -33,18 +33,22 @@ I've used Node v12.14.1
 
 ## How it works
 
-Just input your data and you'll get the result.
+Just input your data and you'll get the result. Note that required fields are marked with an '\*'. After all required fields have been filled, you can copy the signature to the clipboard, or download the signature html file to your computer.
 
 ## Screenshots
 
 Main page
 
-![Main page](screenshots/Main.png?raw=true "Main page")
+![Main page](https://temp-ata-signature-assets.s3.amazonaws.com/main-page.png "Main page")
 
 Signature with logo
 
-![Signature with logo](screenshots/NoPhotoSignature.png?raw=true "Signature with logo")
+![Signature with logo](https://temp-ata-signature-assets.s3.amazonaws.com/signature-with-logo.png "Signature with logo")
```

Comment:
Looks like these screenshots are local, not in the S3 (or remote locations). Should we delete any local /screenshots or should we move the S3 images to the repo?